UITests/Responses/Receipts/PrepaidCardSecondaryResponse.json

```diff
@@ -0,0 +1,63 @@
+{
+    "count": 2,
+    "offset": 0,
+    "limit": 10,
+    "data": [
+        {
+            "token": "trm-token",
+            "primaryCardToken": "trm-0000",
```

Comment:
primary card token should be the token of primary card, and token should be unique, so update data to have unique token each